Pipfile

```diff
@@ -13,6 +13,7 @@ isort = "<6.0" # Unpin when isort==6.0.0 is released
 invenio-app-rdm = {version = "~=12.0.0b2.dev3", extras = ["opensearch2"]}
 invenio-logging = {extras = ["sentry-sdk"], version = "~=1.3.0"}
 zenodo_rdm = {editable="True", path="./site"}
```

Comment:
major/or nullable (if Im wrong): This will appear in the `Pipfile.lock`. Then when we run tests for migrator we doa `pipenv install`, wouldn't it cause issues with incompatible deps?

Reply:
I've added an "if" in the GitHub action to run `pipenv install` only for the `site` test matrix, so we're covered (though a bit "hackish" and arbitrary solution 😅 )